community/kernel/README.md

```diff
@@ -0,0 +1,33 @@
+# Neo4j Kernel
+
+This module, for historical reasons, contains multiple important components of Neo4j:
+
+ - The embedded Java API
+    - org.neo4j.graphdb
+ - The embedded Java API implementation
+    - org.neo4j.kernel.coreapi
+    - org.neo4j.kernel.core
+ - The embedded Traversal Java API
+   - org.neo4j.graphdb.traversal
+ - The embedded Traversal API implementation
+    - org.neo4j.kernel.traversal
+ - Batch Import
```

Comment:
let's not confuse the term `Batch Import` with the old `BatchInserter`, also both could be mentioned:

```
- Batch insertion
  - org.neo4j.unsafe.batchinsert
- Batch import
  - org.neo4j.unsafe.impl.batchimport
```

the manual mentions how they relate to and differentiate from one another.